modules/living_spaces_users/living_spaces_users.permissions.yml

```diff
@@ -4,3 +4,9 @@ access employees overview:
 manage employee profiles:
   title: 'Access living spaces employee profiles'
   description: 'Allow access to manage employee profiles'
+add members to any space:
+  title: 'Add members to any space'
+  description: 'Allow access to add members to any space'
+add members to administered space:
+  title: 'Add members to administered space'
+  description: 'Allow access to add members to administered space'
```

Comment:
These permissions will mostly be used in space-dependant modules, why you have not added these to living_spaces_group instead? it would make more sense as we are talking about members here, not users